pyvlx/node_helper.py

```diff
@@ -18,6 +18,7 @@ def convert_frame_to_node(pyvlx, frame):
                       position_parameter=frame.current_position, rain_sensor=True)
 
     if frame.node_type == NodeTypeWithSubtype.ROLLER_SHUTTER or \
+            frame.node_type == NodeTypeWithSubtype.ADJUSTABLE_SLUTS_ROLLING_SHUTTER_WITH_PROJECTION or \
```

Comment:
Hi @pawlizio ,

in your PR, did you remove `ADJUSTABLE_SLUTS_ROLLING_SHUTTER_WITH_PROJECTION` by intent?

Reply:
Nope, may by a mistake when sorting devices. I tried to group all devices with "orientation of the slats" on functional parameter 3 to Blinds and keep others RollerShutters.

Reply:
But just to make sure, these objects were added by you with changeset `572d0653cf5f941a86aeb024e5871748605c2d19` ...

(note to self: merge earlier to avoid such a mess :-) )